lib/themes/dosomething/paraneue_dosomething/scss/content/_onboarding.scss

```diff
@@ -42,19 +42,25 @@
     cursor: pointer;
   }
 }
-
-.slideshow {
-  //
+.slideshow__carousel {
+  overflow: hidden;
```

Comment:
this was left over from me experimenting with something, i should just remove the entire selector since it wasnt being used